amundsen_application/static/js/interfaces/User.ts

```diff
@@ -0,0 +1,25 @@
+export interface User {
+  display_name: string;
+  profile_url: string;
+};
+
+// Not a good name, not sure if we can consolidate yet
+// TODO: Change to User when ready.
+export interface PeopleUser {
+  email: string;
+  employee_type: string;
+  display_name: string;
+  first_name: string;
+  full_name: string;
+  github_username: string;
+  is_active: boolean;
+  last_name: string;
+  manager_fullname: string;
+  profile_url: string;
+  role_name?: string;
+  slack_id: string;
+  team_name: string;
+  user_id: string;
+};
+
+export type LoggedInUser = PeopleUser & {};
```

Comment:
What does the `& {}` do? I also noticed this on the `Bookmark` type. 

I'm also thinking about just using `Resource` instead of `Bookmark`, which is no different from `own` and `read` resources from a data perspective.

Reply:
It's something that helps for iterating quickly during development. Given that we were at one point less sure about what params we'd be getting from the backend, this structure would let you freely modify `LoggedInUser` by doing:
```
export type LoggedInUser = PeopleUser & { 
   paramFromDummyDataUntilBackendIsReady: ...
}
```
Please feel free to clean up as you work towards completing the Profile Page feature, it's my understanding that the APIs are pretty much set in stone right now.